src/Integration.Vsix/SonarLintIntegrationPackage.cs

```diff
@@ -36,6 +36,7 @@ namespace SonarLint.VisualStudio.Integration.Vsix
     // Register this class as a VS package.
     [PackageRegistration(UseManagedResourcesOnly = true, AllowsBackgroundLoading = true)]
     [Guid(CommonGuids.Package)]
+    [ProvideMenuResource("Menus.ctmenu", 1)]
```

Comment:
Previously this attribute was on the generated partial class. However, it's still required - it notifies VS (or more accurately the VSSDK build targets) that this package handles menu commands.